plugins/leemons-plugin-assignables/frontend/src/components/Details/components/TaskOngoingList/hooks/useTaskOngoingInstanceParser.js

```diff
@@ -50,27 +50,28 @@ function getGradesGraphData(evaluationSystem, students) {
 }
 
 function getStatusGraphData(students) {
+  const [t] = useTranslateLoader(prefixPN('activity_status'));
```

Comment:
Hook inside non-hook, non-component function? 🤔

Reply:
Oh my mistake, i didn't realize. Sorry. I fix it 👌🏼